build/go_check.sh

```diff
@@ -26,6 +26,25 @@ if [[ "${f}" ]]; then
   exit 1
 fi
 
+# Check dependencies up-to-date
+cp go.mod go.mod.prev
+go mod download
+outdated=$(go get -u ./... 2>&1)
+go_mod_diff=$(diff -u go.mod go.mod.prev)
```

Comment:
diff will return 1 if there is diff, so might exit here right away.

Reply:
Yes, you're right, it seems to exit 1 even if stored in a variable. I'll ignore the exit code here.